src/draive/openai/lmm_session.py

```
@@ -1,301 +1,354 @@
+                                    requests: list[LMMToolRequest] = []
+                                    responses: list[LMMToolResponse] = []
+                                    for request, response in pending_tool_calls.values():
+                                        assert response is not None  # nosec: B101
+                                        requests.append(request)
+                                        responses.append(response)
+
+                                    # remember requests and responses
+                                    await memory.remember(
+                                        LMMToolRequests.of(
+                                            requests,
+                                            # TODO: we are missing extra content
```

Pylint (3.3.7)

[warning] TODO: we are missing extra content

(W0511)